app/data/1_0001.json

```diff
@@ -0,0 +1,5127 @@
+{
+    "data_version": "0.0.2",
+    "description": "UK Innovation Survey",
+    "groups": [
+        {
+            "blocks": [
+                {
+                    "id": "general-business-information-block-3",
+                    "sections": [
+                        {
+                            "description": "<p>3 year period 1 January 2014 to 31 December 2016.</p>",
+                            "id": "general-business-information-section-3",
+                            "number": "1",
+                            "questions": [
+                                {
+                                    "answers": [
+                                        {
+                                            "description": "",
+                                            "id": "general-business-information-answer-3-0",
```

Comment:
All answer id's need to be looked at. The id for this should be something like `geographic-markets-answer`